src/mesh/Utils.cpp

```diff
@@ -1,16 +1,24 @@
 #include <Eigen/Core>
 #include <mesh/Edge.hpp>
 #include <mesh/Mesh.hpp>
+#include <mesh/Utils.hpp>
 #include <utils/IntraComm.hpp>
+#include "utils/assertion.hpp"
 
 namespace precice::mesh {
 
 /// Given the data and the mesh, this function returns the surface integral. Assumes no overlap exists for the mesh
 Eigen::VectorXd integrateSurface(const PtrMesh &mesh, const PtrData &data)
 {
-  const int       valueDimensions = data->getDimensions();
+  return integrateSurface(mesh, data->values());
+}
```

Comment:
I assume we could remove these then as well, right?

Reply:
Removed the integration dataid overloads